react/src/components/ManageMapProjectPanel/PublicTabContent.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Project, ProjectRequest } from '@hazmapper/types';
+import { Flex, Button, Card, Typography, Modal } from 'antd';
+import { CopyFilled, CopyOutlined, GlobalOutlined } from '@ant-design/icons';
+import { useAppConfiguration } from '@hazmapper/hooks';
+
+interface PublicTabProps {
+  project: Project;
+  onProjectUpdate: (updateData: Partial<ProjectRequest>) => void;
+  isPending: boolean;
+}
+
+const PublicTabContent: React.FC<PublicTabProps> = ({
+  project,
+  onProjectUpdate,
+  isPending,
+}) => {
+  const config = useAppConfiguration();
+  const togglePublic = () => {
+    onProjectUpdate({ public: !project.public });
+    setIsModalOpen(false);
+  };
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const hostName = window.location.host;
+  const publicPath = `${hostName}/project-public/${project.uuid}`;
```

Comment:
this would be off for things like /staging; i see you got that in line 40 with config.basePath. but maybe just swapping out the `project` path with `project-public`
const { pathname } = window.location;
const publicPath = pathname.replace('/project/', '/project-public/');

Reply:
Ok good catch. I'll switch this, but it makes the copied text working locally not the actual url. I think that will be fine as it only effects local dev.
![Screenshot 2025-02-11 at 5 56 01 PM](https://github.com/user-attachments/assets/463aa36c-4e4a-44a7-b0cf-9abb73278d39)